### PR TITLE
Bump dependencies and Gradle Wrapper, fix deprecations, small build optimization

### PR DESCRIPTION
**Pull request by PatrykMis**

### CODE REVIEW

File: talkback/build.gradle [hunk 1]
```diff
@@ -11,10 +11,8 @@ dependencies {
     implementation project(':brailledisplay')
 }
 
-def talkbackMainPermission = '"com.android.talkback.permission.TALKBACK"'
```
[the remark this answers is not in the document]
Comment (Zanthed) [reply talkback/build.gradle:14]: There are more usages of `talkbackMainPermission` besides here.

**Comment by PatrykMis** [comment]
@Zanthed @thestinger something wrong with this PR?

**Comment by Zanthed** [comment]
Will test this in my own personal fork, though I'm fairly positive there will be a new Talkback code dump when Android 13 comes out.

**Comment by PatrykMis** [comment]
@Zanthed thanks. Is everything ok now? From my point of view and tests it is.

**Comment by Zanthed** [comment]
We need to wait until Android 13. Now is not the time to be merging new content really. Android 13 is less than 2 weeks away and the last thing we need are more possible issues during the Android 12.1 -> 13 port.

After Android 13 is out and finalised for us, we can come back to this. I am highly certain there will be a new Talkback code dump when Android 13 is out.

**Comment by thestinger** [comment]
Applied the Gradle update across all the GrapheneOS standalone apps and AttestationServer so that commit won't be needed.

**Comment by thestinger** [comment]
Also seems to need a rebase.

**Comment by PatrykMis** [comment]
Thanks, done. Everything should be ok now.

**Comment by PatrykMis** [comment]
There is v13.0.0 on Play Store already. I've mentioned @simplexatg on one of pull requests, but updates are not pushed more often than approx. 6 months. Let's hope we see 13.0.0's source code sooner than later.

**Comment by PatrykMis** [comment]
Everything tested on some devices and working except top left back button in TalkBack's settings which is upstream bug (don't know how to fix it).

**Comment by thestinger** [comment]
@PatrykMis Can you rebase this?

**Comment by PatrykMis** [comment]
Yes of course, done.

**Pull request merged**